astropy/io/fits/column.py

```diff
@@ -2402,3 +2414,105 @@ def _convert_ascii_format(format, reverse=False):
             recformat += str(width)
 
         return recformat
+
+
+def _parse_tdisp_format(tdisp):
+    """
+    Parse the ``TDISPn`` keywords for ASCII and binary tables into a
+    ``(format, width, precision, exponential)`` tuple (the TDISP values
+    for ASCII and binary are identical except for 'Lw', which is only present in BINTABLE
+    extensions
+    """
+
+    # Use appropriate regex for format type
+    tdisp = tdisp.strip()
+    if tdisp[0] == 'E':
+        if tdisp[1] in ('N', 'S'):
+            tdisp_re = TDISP_RE_DICT['ENES']
+        else:
+            tdisp_re = TDISP_RE_DICT['EGD']
+    elif tdisp[0] in ('A', 'L'):
+        tdisp_re = TDISP_RE_DICT['A']
+    elif tdisp[0] in ('I', 'B', 'O', 'Z'):
+        tdisp_re = TDISP_RE_DICT['IBOZ']
+    elif tdisp[0] == 'F':
+        tdisp_re = TDISP_RE_DICT['F']
+    elif tdisp[0] in ('G', 'D'):
+        tdisp_re = TDISP_RE_DICT['EGD']
+    else:
+        raise VerifyError('Format {!r} is not recognized.'.format(tdisp))
```

Comment:
May need some thought whether this should be an exception, since we don't want to break the ability to read a fits file just because we cannot display the column exactly as intended!

Reply:
True, but this takes all correct values for TDISPn, so if it doesn't match one of these, it means it's not using a valid FITS value for that keyword...

Reply:
That's true as well, but we should respect the extent to which the user wants to verify the header.